CHANGELOG.rst

```diff
@@ -16,6 +16,10 @@ Changed:
 - Changed error message to Please turn the instrument off, unplug from the PC, and then wait 10 seconds before attempting to use again
 - Enabled option for copying and pasting barcodes
 
+Fixed:
+^^^^^^
+- Fixed heat map display bug by chaniging mean values calculation to the last 5 data points of each well
```

Comment:
Typo is still here `chaniging`. Also since this is in a section titled `Fixed` you don't need to start off the entry with `Fixed heat map` since it's redundant